packages/server/database/rethinkDriver.ts

```diff
@@ -231,7 +231,16 @@ const getRethink = async () => {
   }
   // this is important because pm2 will restart the process & for whatever reason r isn't always healthy
   await r.waitForHealthy()
-  return (r as unknown) as ParabolR
+  return r as unknown as ParabolR
+}
+
+export const closeRethink = async () => {
+  if (promise) {
+    await (await promise)?.drain()
```

Comment:
+1 Nice that you thought of resetting the local state. We can now get rid of the optional chaining `?` here since you check promise the line above.